src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -57,6 +59,10 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
     toggleSidebar();
   }, [toggleSidebar]);
 
+  const goToAdmin = useCallback(() => {
```

Comment:
There are about 4 functions defined with name `goTo*` and 6 functions defined with `navigateTo*`. Could be useful if we settle on one of these conventions while it's easy to track  them.